channels/applicationinsights-channel-js/src/SendBuffer.ts

```diff
@@ -124,6 +129,13 @@ abstract class BaseSendBuffer {
         
                 return null;
             };
+
+            _self.deepCopy = (buffer: BaseSendBuffer) => {
```

Comment:
Suggestion: Lets either
- rename this to `deepCopyTo` 
- reverse the logic so that it copies the passed buffer to this instance (which is the default way I started reading this), as to me deepCopy means either create a deep copy of yourself and return it or create a deepCopy of the passed buffer.
- change this so that it's just `copy` or `clone` or something like that where it returns a new instance etc. (just thinking out aloud)

Reply:
`copyFrom` might be another good name for this